edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -7,6 +7,7 @@
 %define iotedge_confdir %{aziot_confdir}/edged
 %define iotedge_agent_user edgeagentuser
 %define iotedge_agent_uid 13622
+%global debug_package %{nil}
```

Comment:
Are we sure we don't want to build the debuginfo package? It seems useful to have for potential debugging scenarios.

Reply:
If you want to debug it, a developer can remove this line and run the debug build. Otherwise, I think the default should not create any unnecessary artifacts

Reply:
Note: RedHat and Mariner have the debug disabled by default as well.